iModelCore/ecobjects/PublicApi/ECObjects/ECObjects.h

```diff
@@ -191,6 +191,8 @@ enum class ECObjectsStatus
     InvalidConstantUnit,
     ClassTypeNotCorrect,
     InvalidUnit,
+    PruneSchema,

```

Comment:
More consistent with the other items in this enum I'd call this "SchemaIsPruned".

Reply:
Updated the enum value to `SchemaIsPruned`